README.md

```diff
@@ -1,16 +1,52 @@
 <img src="https://cloud.githubusercontent.com/assets/668093/12567089/0ac42774-c372-11e5-97eb-00baf0fccc37.jpg" alt="OpenMRS"/>
 
-# openmrs-addOnManager
+# openmrs Add-on Manager
 
 This repository contains the openmrs-addOnManager OpenMRS Open Web App.
 
-> Add a description of what your app does here.
+> The openmrs-addOnManager is a tool used for uploading, listing, and deleting existing OWAs and modules.
 
 For further documentation about OpenMRS Open Web Apps see
 [the wiki page](https://wiki.openmrs.org/display/docs/Open+Web+Apps+Module).
 
 ## Development
 
+### Local Setup Instructions
+
+```
+# Get the project
+git clone https://github.com/openmrs/openmrs-owa-addonmanager.git
+
+# Move into the project directory
+cd into openmrs-owa-addonmanager
+
+# Install the dependencies
+npm install
+
+# Copy the webpack configuration from webpack.config.sample.js to webpack.config.js
+cp webpack.sample.config.js webpack.config.js
+
+# Locate the 'appdata/owa' directory and type the following command to get the path to the 'appdata/owa' directory and copy it to the clipboard.
+pwd | pbcopy
+
+An example of the path is: /Users/name/downloads/referenceapplication-standalone-2.6.0/appdata/owa
+
+Modify the path to look like: /Users/name/downloads/referenceapplication-standalone-2.6.0/appdata\\owa/
```

Comment:
Why the mixture of forward and back ward slashes in the path above?

Reply:
@dkayiwa If you have a url e.g /Users/millicent/downloads/referenceapplication-standalone-2.6.0/appdata\owa, there will be an error which is "invalid escape characters in strings" since I want to access a folder that has a forward slash in its name. I added an escaping forward slash to solve that.

Reply:
I used the trailing slash because am accessing a directory which is "appdata/owa". When you leave off the slash on a URL that is pointing to a directory, you force the server to do a redirect. While it may seem instantaneous to you, it takes slightly longer for a page to load from a redirect than from the direct URL. Every little bit adds up when it comes to website speed and page load time, so add that slash! Even if it only improves performance a tiny, tiny bit, that is a good thing and taken as a whole, all your little improvements will make a bigger difference.